pages/common/black.md

```diff
@@ -0,0 +1,24 @@
+# black
+
+> A speedy Python auto code formatter.
+> More information: <https://github.com/psf/black>.
+
+- Auto-format a file or entire directory:
+
+`black {source_file_or_directory}`
+
+- Format the code passed in as a string:
+
+`black -c {{source_file_or_directory}}`
+
+- Output a diff for each file on stdout:
+
+`black --diff {{source_file_or_directory}}`
+
+- Don't write the files back, just return the status:
+
+`black --check {{source_file_or_directory}}`
+
+- Don't emit non-error messages to stderr:
```

Comment:
Once again the verb thing: 
```suggestion
- Auto-format a file or directory without emitting non-error messages to stderr:
```

Reply:
That double negation feels awkward, I'd probably suggest:

    - Auto-format a file or directory emitting only error messages to stderr: